pkgs/by-name/ni/nix-init/package.nix

```diff
@@ -45,18 +48,21 @@ rustPlatform.buildRustPackage rec {
     pkg-config
   ];
 
-  buildInputs = [
-    bzip2
-    curl
-    libgit2
-    openssl
-    zlib
-    zstd
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    darwin.apple_sdk.frameworks.Security
-  ] ++ lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isx86_64) [
-    darwin.apple_sdk.frameworks.CoreFoundation
-  ];
+  buildInputs =
+    [
+      bzip2
+      curl
+      libgit2
+      openssl
+      zlib
+      zstd
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      darwin.apple_sdk.frameworks.Security
+    ]
+    ++ lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isx86_64) [
+      darwin.apple_sdk.frameworks.CoreFoundation
+    ];
```

Comment:
Fwiw these should no longer be required now that the darwin SDKs are included by default, if you want to try removing these.